src/style/MystheaUniverse/Pages/CardsReference.qml

```diff
@@ -1,30 +1,76 @@
 import QtQuick 2.14
 import QtQuick.Controls 2.14
 
+import Mysthea.Models 1.0
 import MystheaUniverse.Theme 1.0
 
 Page {
     id: root
     title: qsTr("CARDS REFERENCE")
     objectName: PageName.cardPage
-    property Action leftAction: null
-    property bool hasToolbarLine: true
+
     property color searchFieldBorderColor
+    property Action leftAction: null
+    property Component cardsListComponent: null
+    property TypeProxyModel typeProxyModel: null
+    property TypeComboBoxModel typeComboBoxModel: null
+    property CommandComboBoxModel commandComboBoxModel: null
```

Comment:
Add a note about the models and the fact that these are Mysthea models. The plan is to create an abstract base type which can be extended to create specific models for icaion, mysthea and volfyirion.